corehq/apps/cloudcare/static/cloudcare/js/form_entry/spec/form_ui_spec.js

```diff
@@ -169,6 +169,39 @@ hqDefine("cloudcare/js/form_entry/spec/form_ui_spec", function () {
              assert.equal(form.children()[0].children()[1].children()[0].children()[0].children()[1].children().length, 1); // [q]
         });
 
+        it('Should add n-per-row style to inner Group when using n-per-row-repeat and render Repeat Group and Question grouped by row', function () {
+            let styleObj = {raw: '3-per-row-repeat'};
+
+            let g0 = fixtures.groupJSON({
```

Comment:
Since this is about grouping repeat groups, shouldn't this also insert `repeatJSON` as its children to test that it is correclty grouped?

And I forget, is the intention for `3-per-row-repeat` to also group subgroups? Like if the form builder is 
- Group1 (3-per-row-repeat)
       -Group2
       - Repeat Group1 
       - Repeat Group2
  
"Group2" isn't supposed to be bundled with "Repeat Group1" and "Repeat Group2" right?

Reply:
Leaving a note: no it's not supposed to also group the `Group` nested within `n-per-row-repeat`. It should apply to only `Repeat` questions.